src/main/java/io/appium/java_client/remote/MobileCapabilityType.java

```diff
@@ -45,6 +45,11 @@
      * The kind of mobile device or emulator to use.
      */
     String DEVICE_NAME = "deviceName";
+    
+    /**
+     * The device orientation.
+     */
+    String DEVICE_ORIENTATION = "deviceOrientation";
```

Comment:
How is it different from `orientation` that we already have [here](https://github.com/appium/java-client/blob/master/src/main/java/io/appium/java_client/remote/MobileCapabilityType.java#L93)? I couldn't find one either in server doc's.

Reply:
For example, Sauce uses "deviceOrientation".  Please have a look at the following screenshot:

![image](https://user-images.githubusercontent.com/3188176/34646494-cf29d2d4-f337-11e7-8b0b-6050f3c9377c.png)

Reply:
@MirzaSB We haven't added vendor specific capabilities here so far. Here since we already have `Orientation` it will add confusion regardless of documentation. So two possibilities, adding it with proper java doc comments and other option is to leave it as is. leaving it to @TikhomirovSergey

Reply:
I am not sure that it is the good practice to add and support vendor-specific capabilities to the general project.
@MirzaSB There is nothing prevents you to add the local constant tp your project and use it.